pkg/controller/bootstrap/bootstrapsigner.go

```diff
@@ -159,15 +161,17 @@ func (e *Signer) Run(ctx context.Context) {
 	defer utilruntime.HandleCrash()
 	defer e.syncQueue.ShutDown()
 
-	if !cache.WaitForNamedCacheSync("bootstrap_signer", ctx.Done(), e.configMapSynced, e.secretSynced) {
+	logger := klog.FromContext(ctx)
+	logger.Info("Starting", "controller", e.name)
+	defer logger.Info("Shutting down controller", "controller", e.name)
+
+	if !cache.WaitForNamedCacheSync(e.name, ctx.Done(), e.configMapSynced, e.secretSynced) {
```

Comment:
WaitForNamedCacheSync should be also changed to adhere to the `keysAndValues` logging format:

Reply:
@atiratree The whole package `cache` doesn't support the structured logging yet, so I prefer to do it in a follow-up PR. What do you think?

And it is tracked in the KEP as well: [contextual logging](https://github.com/kubernetes/enhancements/issues/3077)

Reply:
ack, makes sense to do it later